featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -605,3 +608,11 @@ def pd_top3(x):
         else:
             for i1, i2 in zip(true_results.iloc[i], df.iloc[i]):
                 assert (pd.isnull(i1) and pd.isnull(i2)) or (i1 == i2)
+
+
+def _assert_agg_feats_equal(f1, f2):
```

Comment:
would just checking `f1.unique_name() == f2.unique_name()` be enough given the what the method is supposed to do? i don't see any harm in adding the other stuff but unique name should have all this info below included if it is working properly, i think.

Reply:
Generally that should be enough, but in this case that would be `True` even if one of `relationship_path`s is a `list` and the other is a `RelationshipPath`. `RelationshipPath` mostly behaves like a list so most of the logic still works, and the line which triggered the error is only hit under specific conditions:
https://github.com/Featuretools/featuretools/blob/a3966fa3b3f078d237f0f27465d56c5f27778395/featuretools/feature_base/feature_base.py#L617-L621

I could just compare `unique_name` and `relationship_path` – I added the other comparisons to be careful but they don't seem necessary.

Reply:
I see. we can leave it as is for now